### PR TITLE
Draft of 2020 nomination form

### PR DESCRIPTION
**Pull request by afshin**
This is the proposed nomination form for the 2020 cohort of Jupyter Distinguished Contributors.

Please review, comment, and amend as necessary.

### CODE REVIEW

File: nomination-forms/2020.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+
+# Jupyter Distinguished Contributors
+## 2020 JDC Nomination Form
+
+> *Project Jupyter Distinguished Contributors are recognized for their substantial contributions to Jupyter itself in both quality and quantity over at least two years.*
+> 
+> *Contributions may include code, code review, infrastructure work, mailing list and chat participation, community help/building, education and outreach, fundraising, branding, marketing, inclusion and diversity, UX design and research, etc.*
+
+Nominee's name: ______________________________
+
```
Comment (choldgraf) [inline nomination-forms/2020.md:14]:
```suggestion
Nominee's GitHub handle (if applicable): ______________________________

```

**Comment by Carreau** [comment]
I like this, I can think of a few extra information we can gather for the awardees, like prefered pronouns, and alike; but I think we can ask those later. 

I would mention in the form whether the  names of the nominees will be public; I don't believe it is; but I think it is important to say it. 

I would also leave a bit more room for links that that can show typical user contributions, or people may not put links in the 250 words limits ?

**Comment by SylvainCorlay** [comment]
> I would mention in the form whether the names of the nominees will be public; I don't believe it is; but I think it is important to say it.

+1 on making it explicit what is public. For example, I presume that the content of the nomination will not be public even if the person is eventually selected.

**Comment by SylvainCorlay** [comment]
Besides that, it all looks fine to me.

**Comment by damianavila** [comment]
> Besides that, it all looks fine to me.

+1 to me as well.

**Review by Zsailer — APPROVED** [review]

**Review by SylvainCorlay — APPROVED** [review]

File: nomination-forms/2020.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+If relevant, approximately when (month/year) did the nominee's contributions to Jupyter itself end?
+
+To which parts of Jupyter did the nominee directly contribute? Please select up to five areas.
+- [ ] `ipython` (also `ipykernel`, `traitlets`)
+- [ ] Jupyter Core (`nbformat`, `nbconvert`, protocol standards, `jupyter_client`, etc.)
+- [ ] Jupyter Notebook
+- [ ] Jupyter Widgets
+- [ ] JupyterLab
+- [ ] JupyterHub
```
Comment (choldgraf) [inline nomination-forms/2020.md]:
```suggestion
- [ ] JupyterHub or Binder
```

**Review by lresende — APPROVED** [review]

**Comment by afshin** [comment]
We can use the invitation email to ask for nominations to clarify that this information is *not* meant to be public. Thank you all for commenting and feedback!

**Pull request merged**